install.sh

```diff
@@ -137,3 +137,17 @@ for arch in x86 x64 arm arm64; do
     cat msvcenv.sh | sed 's/ARCH=.*/ARCH='$arch/ > bin/$arch/msvcenv.sh
 done
 rm msvcenv.sh
+
+host=x64
```

Comment:
Nice of you to split out the host arch here! FWIW I do have an experimental branch for running on aarch64 (see the branch `arm64` in this repo), so for that branch it's easy to pick the right host arch here too.